Description: Drop dep on loom since it's not packaged
Author: John Goerzen <jgoerzen@complete.org>
Last-Update: 2022-10-31
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -34,18 +34,10 @@
 [dev-dependencies.criterion]
 version = "0.3"
 
-[dev-dependencies.loom]
-version = "0.5"
-features = ["checkpoint"]
-
 [dev-dependencies.proptest]
 version = "1"
 
 [dev-dependencies.slab]
 version = "0.4.2"
-[target."cfg(loom)".dependencies.loom]
-version = "0.5"
-features = ["checkpoint"]
-optional = true
 [badges.maintenance]
 status = "experimental"
